src/server/multi_command_squasher.h

```diff
@@ -85,6 +89,8 @@ class MultiCommandSquasher {
   size_t num_shards_ = 0;
 
   std::vector<MutableSlice> tmp_keylist_;
+
```

Comment:
add a comment: we increase size in one thread and decrease in another. also `current_reply_size_`